apps/docs-snippets2/src/encoding/working-with-bytes.ts

```diff
@@ -0,0 +1,180 @@
+// #region full
+import { randomBytes } from 'crypto';
+import {
+  ArrayCoder,
+  B256Coder,
+  B512Coder,
+  BigNumberCoder,
+  BooleanCoder,
+  EnumCoder,
+  NumberCoder,
+  RawSliceCoder,
+  StdStringCoder,
+  StringCoder,
+  StructCoder,
+  TupleCoder,
+  VecCoder,
+  hexlify,
+} from 'fuels';
+
+// #region working-with-bytes-1
+const u8Coder = new NumberCoder('u8');
+const encodedU8 = u8Coder.encode(255);
+// encodedU8 = new Uint8Array([255]);
+
+const u16Coder = new NumberCoder('u16');
+const encodedU16 = u16Coder.encode(255);
+// encodedU16 = new Uint8Array([0, 255]);
+
+const u32Coder = new NumberCoder('u32');
+const encodedU32 = u32Coder.encode(255);
+// encodedU32 = new Uint8Array([0, 0, 0, 255]);
+
+const u64Coder = new BigNumberCoder('u64');
+const encodedU64 = u64Coder.encode(255);
+// encodedU64 = new Uint8Array([0, 0, 0, 0, 0, 0, 0, 255]);
+
+const u256Coder = new BigNumberCoder('u256');
+const encodedU256 = u256Coder.encode(255);
+// encodedU256 = new Uint8Array([
+//     0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 255
+// ]);
+// #endregion working-with-bytes-1
+
+console.assert(
+  encodedU8.toString() === new Uint8Array([255]).toString(),
+  'Encoded U8 should be equal to 255'
+);
+console.assert(
+  encodedU16.toString() === new Uint8Array([0, 255]).toString(),
+  'Encoded U16 should be equal to 255'
+);
+console.assert(
+  encodedU32.toString() === new Uint8Array([0, 0, 0, 255]).toString(),
+  'Encoded U32 should be equal to 255'
+);
+console.assert(
+  encodedU64.toString() === new Uint8Array([0, 0, 0, 0, 0, 0, 0, 255]).toString(),
+  'Encoded U64 should be equal to 255'
+);
+console.assert(
+  encodedU256.toString() ===
+    new Uint8Array([
+      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+      255,
+    ]).toString(),
+  'Encoded U256 should be equal to 255'
+);
```

Comment:
Is it worth moving these asserts alongside the calls, then we can remove the comments.

Example:
```ts
const u8Coder = new NumberCoder('u8');
const encodedU8 = u8Coder.encode(255);

console.assert(
  encodedU8.toString() === new Uint8Array([255]).toString(),
  'Encoded U8 should be equal to 255'
);
```

Reply:
7f38a0cb5